Networking/Networking/Model/Refund/OrderItemRefund.swift

```diff
@@ -0,0 +1,163 @@
+        let totalTax = try container.decode(String.self, forKey: .totalTax)
+
+        // initialize the struct
+        self.init(itemID: itemID, name: name, productID: productID, variationID: variationID, quantity: quantity, price: price, sku: sku, subtotal: subtotal, subtotalTax: subtotalTax, taxClass: taxClass, taxes: taxes, refundTotal: refundTotal, totalTax: totalTax)
+    }
+
+    /// The public encoder for OrderItemRefund.
+    ///
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: EncodingKeys.self)
+
+        try container.encode(itemID, forKey: .itemID)
+        try container.encode(name, forKey: .name)
+        try container.encode(productID, forKey: .productID)
+        try container.encode(variationID, forKey: .variationID)
+        try container.encode(Double(truncating: quantity), forKey: .quantity)
+        try container.encode(price.stringValue, forKey: .price)
+        try container.encode(sku, forKey: .sku)
+        try container.encode(subtotal, forKey: .subtotal)
+        try container.encode(subtotalTax, forKey: .subtotalTax)
+        try container.encode(taxClass, forKey: .taxClass)
+
+        try container.encode(taxes, forKey: .taxes)
+
+        try container.encode(refundTotal, forKey: .refundTotal)
+        try container.encode(totalTax, forKey: .totalTax)
+    }
+}
+
+
+/// Defines all of the OrderItemRefund CodingKeys.
+///
+private extension OrderItemRefund {
+
+    enum EncodingKeys: String, CodingKey {
+        case itemID         = "id"
+        case variationID    = "variation_id"
+        case name
+        case productID      = "product_id"
+        case quantity
+        case price
+        case sku
+        case subtotal
+        case subtotalTax    = "subtotal_tax"
+        case taxClass       = "tax_class"
+        case refundTax      = "refund_tax"
+        case refundTotal    = "refund_total"
+        case totalTax       = "total_tax"
+        case taxes
+    }
+
+    enum DecodingKeys: String, CodingKey {
+        case itemID         = "id"
+        case variationID    = "variation_id"
+        case name
+        case productID      = "product_id"
+        case quantity
+        case price
+        case sku
+        case subtotal
+        case subtotalTax    = "subtotal_tax"
+        case taxClass       = "tax_class"
+        case refundTax      = "refund_tax"
+        case refundTotal    = "total"
+        case totalTax       = "total_tax"
+        case taxes
+    }
+}
+
+
+// MARK: - Comparable Conformance
+//
+extension OrderItemRefund: Comparable {
+    public static func == (lhs: OrderItemRefund, rhs: OrderItemRefund) -> Bool {
+        return lhs.itemID == rhs.itemID &&
+            lhs.name == rhs.name &&
+            lhs.productID == rhs.productID &&
+            lhs.quantity == rhs.quantity &&
+            lhs.price == rhs.price &&
+            lhs.sku == rhs.sku &&
+            lhs.subtotal == rhs.subtotal &&
```

Comment:
This is a genuine question. Do we need to use tax info to compare refunds as well? Is not item ID, product ID and variation id enough? I am trying to fully understand the potential use cases of a refund

Reply:
Yeah, the IDs are probably enough. OrderItemRefunds aren't going to be as ubiquitous as a Product (for example).